README.md

```diff
@@ -9,7 +9,7 @@ Python and JavaScript. It aims to be a drop-in replacement for
 See [posativ.org/isso](http://posativ.org/isso/) for more details.
 
 ## License
-MIT, see [LICENSE](LICENSE).
+MIT, see [LICENSE](https://github.com/posativ/isso/blob/master/LICENSE).
```

Comment:
This breaks things for forks, can we just refer to the local file?

Reply:
You cannot link to package resources inside PyPi, even if you include them in `MANIFEST.in`.
(I tested via https://test.pypi.org/project/isso-testpypi/ and the link goes to a 404, as you'd expect: https://test.pypi.org/project/isso-testpypi/LICENSE )